src/pages/admin/training/list/components/spec/trainingTable.spec.tsx

```diff
@@ -1,78 +1,123 @@
 
   });
 
-  it('Should display trainings tabular data', () => {
-    // Arrange
-    const trainings: TrainingSummary[] = [
-      {
-        id: 2,
-        isActive: true,
-        name: 'John Doe',
-        start: new Date(),
-        end: new Date(),
-      },
-      {
-        id: 3,
-        isActive: false,
-        name: 'Mark Somez',
-        start: new Date(),
-        end: new Date(),
-      },
-    ];
+  it('Should return a react-virtualized Table', () => {
+     // Arrange
+    const trainingList = [];
+    const width = 0;
 
     // Act
     const trainingTableComponent = shallow(
-      <TrainingTableComponent trainingList={trainings}/>,
+      <TrainingTableComponent width={width} trainingList={trainingList}/>,
     );
 
-    // Assert
-    const expectedDomTree = `
-      <table>
-        <tbody>
-          <tr>
-            <td>
-              <span>
-                2
-              </span>
-            </td>
-            <td>
-              <span>
-                John Doe
-              </span>
-            </td>
-          </tr>
-          <tr>
-            <td>
-              <span>
-                3
-              </span>
-            </td>
-            <td>
-              <span>
-                Mark Somez
-              </span>
-            </td>
-          </tr>
-        </tbody>
-      </table>
-      `;
-
-    expect(trainingTableComponent.html()).to.be.equal(multilineTrim(expectedDomTree));
+    // Asser
+    expect(trainingTableComponent.type()).to.be.equals(Table);
+  });
+
+  describe('header tests', () => {
+    it('should render an header column to show the "isActive" training property', () => {
+       // Arrange
+      const trainingList = [];
+      const width = 100;
+
+      // Act
+      const trainingTableComponent = shallow(
+      <TrainingTableComponent width={width} trainingList={trainingList}/>,
+      );
+      const headerColumns = trainingTableComponent.children();
+
+      // Assert
+      expect(headerColumns.length).to.be.equals(3);
+      expect(headerColumns.at(0).prop('label')).to.be.equals('Active');
+      expect(headerColumns.at(0).prop('dataKey')).to.be.equals('isActive');
+      expect(headerColumns.at(0).prop('width')).to.be.equals(10); // 10% of 100px
+    });
+
+    it('should render an header column to show the "name" training property', () => {
+       // Arrange
+      const trainingList = [];
+      const width = 100;
+
+      // Act
+      const trainingTableComponent = shallow(
+      <TrainingTableComponent width={width} trainingList={trainingList}/>,
+      );
+      const headerColumns = trainingTableComponent.children();
+
+      // Assert
+      expect(headerColumns.length).to.be.equals(3);
+      expect(headerColumns.at(1).prop('label')).to.be.equals('Name');
+      expect(headerColumns.at(1).prop('dataKey')).to.be.equals('name');
+      expect(headerColumns.at(1).prop('width')).to.be.equals(70); // 10% of 100px
+    });
```

Comment:
Pending to add the third column. I suppose that there are 3 because `expect(headerColumns.length).to.be.equals(3);`